roadmap/4_governance_nns/in-progress/1_sns_token_minting.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Ability to mint SNS tokens and revised thresholds for voting
+links: https://forum.dfinity.org/t/new-sns-ability-to-mint-sns-tokens-revised-thresholds-for-voting/23382
+  Forum Post: 
+eta: Q4 23
```

Comment:
```suggestion
eta: Q1 24
```

Looks like it didn't make it into the cut.. Should we thus update this or is it close enough?